pkgs/development/libraries/irr1/default.nix

```diff
@@ -4,25 +4,25 @@
 , cmake
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (self: {
   pname = "irr1";
   version = "1.9.4";
 
   src = fetchFromGitHub {
     owner = "berndporr";
     repo = "iir1";
-    rev = version;
+    rev = self.version;
     hash = "sha256-T8gl51IkZIGq+6D5ge4Kb3wm5aw7Rhphmnf6TTGwHbs=";
   };
 
   nativeBuildInputs = [ cmake ];
 
   meta = {
-    description = "A DSP IIR realtime filter library written in C++";
-    downloadPage = "https://github.com/berndporr/iir1";
     homepage = "http://berndporr.github.io/iir1/";
+    description = "A DSP IIR realtime filter library written in C++";
```

Comment:
Why is the order changed in this way?

Reply:
Mostly personal style. I put homepage first on meta attrs.

Reply:
I don't personally agree with the change, but fair enough